backend/infrahub/core/validators/uniqueness/query.py

```diff
@@ -137,12 +137,11 @@ async def query_init(self, db: InfrahubDatabase, **kwargs: Any) -> None:  # noqa
         # ruff: noqa: E501
         query = """
         // get attributes for node and its relationships
-        CALL {
+        CALL () {
```

Comment:
it feels like this shouldn't be empty, most likely we need to investigate what is generated as part of `select_subqueries_str`

Reply:
This is legit: the whole query starts with a subquery, so there are no variables to import at this stage.